source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Raspberry Pi All-In-One Installer"
+date: 2016-05-12 01:39
+comments: true
+sharing: true
+footer: true
+---
+Easily deploy a complete Home-Assistant server, complete with Websocket MQTT and Z-Wave support out of the box using Fabric!
+
+**Usage:**
+*Start with a fresh Raspbian Jessie/Jessie-Lite or Debian 8 install*
```

Comment:
possibly, if some of the internals are in place already, the fabric script will error and die. Eventually it will be more robust and provide other feature sets beyond a fresh server install. The Debian 8 requirement is related to Systemd which ships with 8 by default.